File: back-end/Models/Suggestion.cs
namespace Stockaccino.Models;

public class Suggestion
{
    public string Name { get; set; }

    public string Symbol { get; set; }

    public double Change { get; set; }

    public double ChangePercent { get; set; }

    public double Ask { get; set; }

    public Suggestion(string name, string symbol, double change, double changePercent, double ask)
    {
        this.Name = name;
        this.Symbol = symbol;
        this.Change = change;
        this.ChangePercent = changePercent;
        this.Ask = ask;
    }
}

File: front-end/src/app/classes/yahoo/suggestion.ts
export interface Screener {
  Title: string;
  Description: string;
  Suggestions: Suggestion[];
}

export interface Suggestion {
  Name: string;
  Symbol: string;
  Change: number;
  ChangePercent: number;
  Ask: number;
}

export const screenersCategories: string[] = [
  'day_gainers',
  'day_losers',
  'growth_technology_stocks',
  'undervalued_growth_stocks',
  'most_shorted_stocks',
  'most_actives',
  'undervalued_large_caps',
  'aggressive_small_caps',
  'small_cap_gainers',
  // 'top_mutual_funds',
  'portfolio_anchors',
  'solid_large_growth_funds',
  'solid_midcap_growth_funds',
  'conservative_foreign_funds',
  'high_yield_bond',
];
